### DISCUSSION TITLE
项目启动过程中 node-sass 报错

### DISCUSSION THREAD

**Discussion by djknight1** [alibaba/formily#1260] [Q&A 2.x]
- 问题：
![image](https://user-images.githubusercontent.com/33688921/115382990-e081b600-a207-11eb-9a2e-eed78d2cbcf3.png)

- package.json
![image](https://user-images.githubusercontent.com/33688921/115383065-f7c0a380-a207-11eb-98fd-b1ed737786f6.png)

**Comment by janryWang** [comment]
试试用postcss-loader
